dev/Repeater/APITests/ViewportTests.cs

```diff
@@ -348,6 +348,13 @@ public void ValidateSupportForScrollerConfigurationChanges()
                 return;
             }
 
+            // Post Rs5 configuration changes will not be raised.
+            if (PlatformConfiguration.IsOsVersionGreaterThanOrEqual(OSVersion.Redstone5))
+            {
+                Log.Warning("Skipping ValidateSupportForScrollerConfigurationChanges");
```

Comment:
In the logs this will be indistinguishable from the above Warning. It'd be better if the log message contained more identifiable information.